configsuite/config.py

```diff
@@ -280,15 +280,15 @@ def _build_initial_merged_config(self, layers, schema):
 
         if isinstance(data_type, configsuite.types.BasicType):
```

Comment:
Could this also be solved by:

```python
if cond1:
    merged = layers[-1]
elif cond2:
    merged = self._build1()
elif cond3:
    merged = ...
else:
    raise SuperError("Hei hei")

return merged
```

After reading on the internet my impression is that the motivation of the error is to lower the complexity of conditional statements by not having a bunch of flow-disrupting statements (like `return` and `raise`) within them in addition. Which kind of makes sense 🤷 And if so, do we like the collecting merged value in a variable and single return better?